contrib/abduco_completion.bash

```diff
@@ -0,0 +1,26 @@
+
+function _abduco() {
+        case $2 in
+        -*) # Option
+                COMPREPLY=($(compgen -W ' \
+                        -n -nf \
+                        -c -cf -cr \
+                        -f -fn -fc -fcr \
+                        -A -Ar \
+                        -a -ar \
+                        -l -lr \
+                        -r -rc -rcf -rA -ra -rl \
+                        -e -e^ \
+                        -v \
+                        ' -- $2))
+        ;;
+        *) # Session
+                local sessions=$(abduco | tail -n+2 | cut -f 3)
```

Comment:
The session name is in the third column for the latest release (`v0.6`) but on the `master` branch it's in the fourth column. `awk '{print $NF}'` (=last column) works for both versions:

```suggestion
                local sessions=$(abduco | tail -n+2 | awk '{print $NF}')
```